### PR TITLE
Adds Editor component

### PR DESCRIPTION
**Pull request by gabrielpoca**
Also fixes small layout issues on the comment form.

There is no carrot color for now, there is no right way to do it, although there are hacks, we can explore them later.

screen shot 2015-01-02 at 17 58 51

### CODE REVIEW

File: app/assets/javascripts/app/components/comment_form/templates/comment_form.html.slim [hunk 1]
```diff
@@ -1,6 +1,6 @@
 .TopBarLayout
   .TopBarLayout-header
-    md-toolbar.Toolbar.Toolbar--clean
+    md-toolbar.md-whiteframe-z1.Toolbar.Toolbar--clean
```
Comment (joaojusto) [inline app/assets/javascripts/app/components/comment_form/templates/comment_form.html.slim]: I think this should't have any shadow.

File: app/assets/javascripts/app/discussions/show.html.slim [hunk 2]
```diff
@@ -1,6 +1,6 @@
 .TopBarLayout
   .TopBarLayout-header
-    md-toolbar.Toolbar.Toolbar--clean
+    md-toolbar.md-whiteframe-z1.Toolbar.Toolbar--clean
```
Comment (joaojusto) [inline app/assets/javascripts/app/discussions/show.html.slim]: I think this should't have any shadow

File: app/assets/javascripts/app/components/comment_form/templates/comment_form.html.slim (hunk 1 above)
Comment (joaojusto) [inline app/assets/javascripts/app/components/comment_form/templates/comment_form.html.slim]: I think this header shouldn't have shadow

Comment (gabrielpoca) [inline app/assets/javascripts/app/components/comment_form/templates/comment_form.html.slim]: There is a shadow https://trello.com/c/Sj9eGWbf/37-fix-textarea-input-on-new-commnet-screen

**Comment by gabrielpoca** [comment]
@joaojusto have a look now

**Comment by joaojusto** [comment]
:+1:

**Pull request merged**